extension/content.js

```diff
@@ -263,6 +263,25 @@ function addPatchDiffLinks() {
 	`);
 }
 
+// Prompt user to confirm erasing a comment with the Cancel button
+$(document).on('click', function (event) {
+	// Check event.target instead of using a delegate, because Sprint doesn't support them
```

Comment:
Not really. We'd only remove stuff we'd never need like ajax/events/etc.

I tried though, with `grunt custom:-ajax,-deprecated,-effects,-sizzle,-exports/amd` in the jquery repo and it still ended up being 50 KB...